.github/workflows/main.yml

```diff
@@ -355,12 +355,12 @@ jobs:
               -p wasmtime --no-default-features --features wat
               -p wasmtime --no-default-features --features profiling
               -p wasmtime --no-default-features --features cache
-              -p wasmtime --no-default-features --features async
+              -p wasmtime --no-default-features --features async,std
```

Comment:
Was this necessary because it was otherwise broken? Or just keeping the same behavior tested?

If broken, could `wasmtime` enable the `std` feature as part of the async feature for now? (until that's fixed)

Otherwise if just keeping the same behavior mind also adding `-p wasmtime-fiber --features std` and `-p wasmtime-fiber` to this matrix?

Reply:
I think I had added it at some point while debugging failures but indeed it should work with just `async`, no `std` -- so, restored that, added `std` separately, and added the `wasmtime-fiber` variants.